feature-utils/poly-look/src/react-components/index.js

```diff
@@ -1,6 +1,7 @@
-import ChipGroup from "./chipGroup/chipGroup.jsx";
+import Chip from "./chip/chip.jsx";
+import FilterChips from "./filterChips/filterChips.jsx";
 import { Tab, Tabs } from "./tabs.jsx";
 import BlockLegend from "./legends/blockLegend/blockLegend.jsx";
 import LineLegend from "./legends/lineLegend/lineLegend.jsx";
 
-export { Tab, Tabs, ChipGroup, BlockLegend, LineLegend };
+export { Chip, Tab, Tabs, FilterChips, BlockLegend, LineLegend };
```

Comment:
If this is merged before #741, it's going to cause conflicts there, and the opposite. Can you maybe review that while you're doing this?

Reply:
The `Chip` and `FilterChips` component are already in the `main` branch, so this shouldn't be here. I have an idea why this is happening though...

Reply:
I just wanted to trick you into reviewing #741. I guess that didn't work :man_shrugging:

Reply:
😆 I will!